recipes/default.rb

```diff
@@ -17,15 +17,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+# Install base IIS in powershell mode until chef client 14, because -
```

Comment:
DISM should be the default in both Chef 13 and 14 so I'm a bit confused about this statement. https://github.com/chef/chef/blob/master/lib/chef/resource/windows_feature.rb#L82